community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/tracing/LogAppendEvent.java

```diff
@@ -25,8 +25,7 @@
  */
 public interface LogAppendEvent extends LogForceEvents, AutoCloseable
 {
-
-    LogAppendEvent NULL = new LogAppendEvent()
+    class Null implements LogAppendEvent
```

Comment:
Ok i found that in BatchingTransactionAppenderConcurrencyTest we extend newly introduced null class, can we please rename it to `Empty` then?